styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -527,7 +539,7 @@ public Subparser parser(Subparsers subCommands) {
   private enum BackfillCommand {
     LIST("ls", "List backfills"),
     CREATE("", "Create a backfill"),
-    DELETE("rm", "Delete a backfill"),
+    HALT("h", "Halt a backfill"),
```

Comment:
`h` was used earlier for `halt` in the workflow/partition case. We had the same discussion at the time and landed in that `h` was ok. https://github.com/spotify/styx/pull/5#pullrequestreview-9744589